jmxtrans-output/jmxtrans-output-elastic/src/main/java/com/googlecode/jmxtrans/model/output/elastic/ElasticWriter.java

```diff
@@ -93,17 +97,20 @@ public ElasticWriter(
 						DEFAULT_ROOT_PREFIX);
 
 		this.connectionUrl = connectionUrl;
+        	this.username = username;
+        	this.password = password;
 		this.indexName = this.rootPrefix + "_jmx-entries";
-		this.jestClient = createJestClient(connectionUrl);
+		this.jestClient = createJestClient(connectionUrl, username, password);
 	}
 
-	private JestClient createJestClient(String connectionUrl) {
+	private JestClient createJestClient(String connectionUrl, String username, String password) {
 		log.info("Create a jest elastic search client for connection url [{}]", connectionUrl);
 		JestClientFactory factory = new JestClientFactory();
 		factory.setHttpClientConfig(
-				new HttpClientConfig.Builder(connectionUrl)
-						.multiThreaded(true)
-						.build());
+                new HttpClientConfig.Builder(connectionUrl)
+                        .defaultCredentials(username, password)
```

Comment:
defaultCredentials() should be configured only if they are not null. We still need to be able to connect anonymously if needed.